tools/buildx-gen.sh

```diff
@@ -66,6 +66,7 @@ target "$image-$variant" {
       proxy_version = "istio-proxy:${PROXY_REPO_SHA}"
       istio_version = "${VERSION}"
     }
+    platforms = ["linux/arm/v7"]
```

Comment:
Agreed, this should neither be hardcoded, I do however want to parameterize this field although will need some assistance on a best possible solution.

Reply:
@sdake perhaps you might have a suggestion on this front? I was considering we could use the `${TARGET_OS} & ${TARGET_ARCH}` values and have them passed into this file?